idl/chromadb/proto/query_executor.proto

```diff
@@ -6,9 +6,15 @@ import "chromadb/proto/chroma.proto";
 
 message ScanOperator {
     Collection collection = 1;
+    // Deprecated
```

Comment:
nit: add a bit more information as to why it has been deprecated and what is the new course of action?

Reply:
This will be removed in the next PR